javascripting/array-filtering.js

```diff
@@ -0,0 +1,6 @@
+var numbers = [1,2,3,4,5,6,7,8,9,10];
+var filtered = numbers.filter(function evenNumbers(number){
```

Comment:
Not exactly, JS like some other languages embraces the idea of anonymous functions or lambdas which are functions without identifiers. They can be passed around like normal variables. So in the case where maybe you're passing in some callback function that's really large, you can just use an anonymous function instead.

This would have also worked...
```
function evenNumbers(num) {
  //lets say your doing a bunch of 
  //other stuff here.
  return n % 2 == 0;
}

numbers.filter(evenNumbers)
```

but because your function body is small, this could be preferable.

```
numbers.filter(function(num) {
  return n % 2 == 0;
});
```